app/code/Magento/AwsS3/Driver/AwsS3.php

```diff
@@ -89,8 +88,8 @@ public function test(): void
     {
         try {
             $this->adapter->write(self::TEST_FLAG, '', new Config(self::CONFIG));
-        } catch (Exception $exception) {
-            throw new DriverException(__($exception->getMessage()), $exception);
+        } catch (\League\Flysystem\FilesystemException $e) {
```

Comment:
I believe it's better to catch `Exception` as well, to be sure (applicable for all other catch blocks in this class)

Please move `League\Flysystem\FilesystemException` to the `use` block

Reply:
Done.